crux-bench/time-series-bench/project.clj

```diff
@@ -0,0 +1,9 @@
+(defproject time-series-bench "0.1.0"
```

Comment:
nit: I know this isn't getting deployed to mvn, but let's give it a proper group-id (not `time-series-bench/time-series-bench` - I _hate_ that lein does this by default)

`juxt/crux-bench`?